pages/specification/json-hyper-schema/_index.md

```diff
@@ -13,6 +13,531 @@ In essence:
 
+
+## Hyper-schema use cases
+
+### Basic concept
+
+**Get a JSON instance**
+
+```http
+GET https://example.com/book/12345
+```
+
+_This `GET` request fetches information about a specific book, identified by the `book` ID (`12345`). In response:_
+
+```http
+200 OK
+Content-Type: application/json
+Link: <https://example.com/schemas/book>; rel="describedby"
+
+{
+  "title": "Mobby-Dick",
+  "authorId": 100,
+  ...
+}
+```
+
+_The server responds with a successful `200 OK` status, JSON-formatted data, and a `Link` header that references the schema describing the structure of the returned data, including details like the `book` `title` and a link to the author via `authorId`._
+
+**Get the schema that describes the instance using the "`describedby`" Link header.**
+
+```http
+GET https://example.com/schemas/book
+```
+
+_This request fetches the schema describing the structure of the `book` object.
+In response:_
+
+```http
+200 OK
+Content-Type: application/schema+json
+
+{
+  "links": [
+    {
+      "href": "/author/{authorId}",
+      "rel": "author",
+      "templateRequired": ["authorId"]
+    }
+  ]
+}
+```
+
+_The response contains a JSON Schema describing the instance's structure and links to relevant resources, such as the author's resource via `rel="author`." It also specifies that the `authorId` is required to complete the URL template._
+
+**Use the LDO combined with the instance to construct a link to get information about the author.**
+
+```http
+GET https://example.com/author/100
+```
+
+This request fetches data for the author with `authorId` 100. In response:
+
+```http
+200 OK
+Content-Type: application/json
+Link: <https://example.com/schemas/author>; rel="describedby"
+
+{
+  "name": "Herman Melville",
+  ...
+}
+```
+
+_The response includes a JSON object with the author's details, and the `Link` header points to the schema that describes the author, following the same format as the book schema._
+
+Keep following links to discover more content.
+
+### Actions
+
+#### Search
+
+Using a link to construct a URI from user input. (Analogous to `<form method="get">`)
+
+```json
+{
+  "links": [
+    {
+      "href": "/books"
+      "rel": "search",
+      "hrefSchema": {
+        "type": "object",
+        "properties": {
+          "title": { "type": "string" },
+          "author": { "type": "integer" },
+          ...
+        }
+      }
+    }
+  ]
+}
+```
+
+```http
+GET https://example.com/books?title=Moby-Dick
+```
+
+_This `GET` request searches for books with the title `Moby-Dick`._
+
+```http
+200 OK
+Content-Type: application/json
+Link: <https://example.com/schemas/books>; rel="describedby"
+
+[
+  {
+    "title": "Mobby-Dick",
+    "authorId": 100,
+    ...
+  }
+]
+```
+
+_The response returns a successful `200 OK` JSON-formatted data with a `Link` to the schema describing the book collection, and the response body contains an array of books, each with details such as the `title` and `authorId`._
```

Comment:
suggestion: add specificity to the books returned in the response. For example:
"...the response returns an array of books whose `title` value is Moby-Dick"